src/app/store/transactions/token-transfer.hooks.ts

```diff
@@ -99,29 +79,16 @@ export function useStxTokenTransferUnsignedTxState(values?: StacksSendFormValues
   return tx.result;
 }
 
-export function useGenerateFtTokenTransferUnsignedTx(
-  assetBalance: StacksFungibleTokenAssetBalance
-) {
+export function useGenerateFtTokenTransferUnsignedTx(asset: Sip010AccountCryptoAssetWithDetails) {
```

Comment:
If `useGenerateFtTokenTransferUnsignedTx` only uses `asset.info`, why do we pass it the whole object?

Reply:
Good point, will check these hooks.